Assets/Scripts/PointerScript.cs

```diff
@@ -687,6 +700,18 @@ public void SetPressure(float fPressure)
             m_CurrentPressure = fPressure;
         }
 
+        public float GetPressure()
+        {
+            return m_CurrentPressure;
+        }
+
+        // utillity that maps one range into another
+        float Remap(float value, float from1, float to1, float from2, float to2)
```

Comment:
Rather tempted to move this to a general utils class. It's so darn useful. Any chance there's already a remap in the codebase we can reuse or factor out?

Reply:
Where else is this required? It's best to leave things as specific, and private unless there is repetition, otherwise the math library will contain just about every math function and 1-2 usages. Someone changes the math instance and breaks something subtle elsewhere.